src/applications/claims-status/tests/components/evss/SubmittedTrackedItem.unit.spec.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+import { expect } from 'chai';
+
+import SubmittedTrackedItem from '../../../components/evss/SubmittedTrackedItem';
+
+describe('<SubmittedTrackedItem>', () => {
+  it('should render submitted item with no docs', () => {
+    const item = {
+      trackedItemId: 1,
```

Comment:
This is confusing. Was the `SubmittedTrackedItem` data updated to use `id` instead of `trackedItemId`?

Reply:
It was, but this is me copying the old unit tests that worked with the EVSS version of claims into a separate file so I can update the existing unit tests to reflect the changes made in this PR